library_management_system_downloader/downloader_configuration_file_TEMPLATE.py

```diff
@@ -0,0 +1,18 @@
+# These are parameters that stay the same from query to query (unlike the
+# doi):
+static_parameters_for_api = {}
+
+user_agent_custom_string = (
+        'This is part of a bulk download for a research '
+        'project by Jane Doe of the University of ________. '
+        'I have put a threshold on the download in order '
+        'not to cause a problem with your servers. If this '
+        'download IS causing a problem, please feel free to email me at '
+        '________, or to call me directly at ________.')
+
+api_base_url = (
+            'https://example.com/openurl')
+
+tsv_dataset_location = '/path/to/dois.tsv'
```

Comment:
Is this the query set or the output set? Let's use a variable name that makes that clear? Also would relative path be more appropriate (all paths within the repo should be relative)

Reply:
I've made the variable name clearer in 06b0fc, by renaming it `input_tsv_dataset_location` and adding a comment above it.